NeuroID/DataStore.swift

```diff
@@ -28,6 +29,24 @@ public enum DataStore {
     }
 
     static func cleanAndStoreEvent(screen: String, event: NIDEvent, storeType: String) {
+        // If we hit a low memory event, drop events and e1arly return.
+        if (NeuroID.lowMemory) {
+            return
+        }
+        // If queue has more than 2000 events, send a queue full event and return
+        if (DataStore.queuedEvents.count + DataStore.events.count >= max_event_size) {
```

Comment:
With the max event size as 1999 this check would become > max_event_size (w/o =)